src/components/modules/CodeRecovery/CodeRecovery.interface.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <code-recovery
+    :email="email"
+    @recovered="onRecovered"
+    @recovery-cancel="onCancel"
+  />
+</template>
+
+<script>
+import CodeRecovery from './CodeRecovery.interactor';
```

Comment:
Honestly - i dont understand which component we are using in template. Propose use components in templates with it's layer. In this case, for example, would be `<code-recorery-interactor>`

Reply:
It should be much obvious for everyone

Reply:
it's not so easy as expected, because `interactor` can be replaced to other `CodeReview` modules and you need replace all any other usage, that's why naming as is